Sources/Error Handling/UnexpectedBackendResponseSubErrorCode.swift

```diff
@@ -1,62 +0,0 @@
-//
-//  Copyright RevenueCat Inc. All Rights Reserved.
-//
-//  Licensed under the MIT License (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//      https://opensource.org/licenses/MIT
-//
-//  UnexpectedBackendResponseSubErrorCode.swift
-//
-//  Created by Joshua Liebowitz on 10/25/21.
-
-import Foundation
-
-enum UnexpectedBackendResponseSubErrorCode: Int, Error {
```

Comment:
These are inside `BackendError` now, and encoded as `BackendError.unexpectedBackendResponse`.